eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxWits.hs

```diff
@@ -231,8 +220,8 @@ instance (Era era, Script era ~ AlonzoScript era) => Semigroup (AlonzoTxWits era
   (<>) x y | isEmptyTxWitness x = y
   (<>) x y | isEmptyTxWitness y = x
   (<>)
-    (getMemoRawType -> AlonzoTxWitsRaw a b c d (Redeemers' e))
-    (getMemoRawType -> AlonzoTxWitsRaw u v w x (Redeemers' y)) =
+    (getMemoRawType -> AlonzoTxWitsRaw a b c d (Redeemers e))
```

Comment:
the master delouser at work!

Reply:
Thank you! Today I learned a new word :smile: